File: Prob48/Solution.cpp
// Approach 1:-

class Solution
{
public:
    int lengthOfLastWord(string s)
    {
        int i = s.size() - 1;

        while (i >= 0 && s[i] == ' ')
            i--;

        int end = i;

        while (i >= 0 && isalpha(s[i]))
            i--;

        return end - i;
    }
};

File: Prob48/Solution.java
// Approach 1:-

class Solution {
    public int lengthOfLastWord(String s) {
        int i=s.length()-1;
        
        while(i>=0 && s.charAt(i)==' ')
            i--;
        
        int end=i;
        
        while(i>=0 && s.charAt(i)!=' ')
            i--;
        
        return end-i;
    }
}